R/pkg/R/mllib.R

```diff
@@ -197,11 +197,10 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
   invisible(x)
   }
 
-#' Make predictions from a generalized linear model
-#'
 #' Makes predictions from a generalized linear model produced by glm() or spark.glm(),
 #' similarly to R's predict().
 #'
+#' @title predict
```

Comment:
I think people have been trying to be consistent within the given R source file but total agree there are many different forms in used and it would be nice to have a single format for all R sources.